API.md

```diff
@@ -2,56 +2,54 @@
 
 ### `void begin()`
 
-Initializes the given interface, prepares the board for communication. All power switches are set to turn-off state.
+Initializes the given interface, prepares the board for communication. All power switches are set to the turned-off state.
 
-Important, call this method at least once, in your `setup()`.
+Call this method before interfacing with Octofet. For example, in your `setup()`.
 
 ### `void digitalWrite(uint8_t channel, bool value, uint8_t device = 0)`
 
-Sets the state ("on" or "off") of one power switch. Function arguments:
+Sets the state ("on" or "off") of one power switch.
 
-- `channel`: ordinal number power switch on device. Range is from `0` to `7`. Use option value is `ALL` to selected and sets state of all power switches.
-- `value`: states selected power switch. Values (turn-on or turn-off) may be switch is different data types: `HIGH` or `LOW` / `true` or `false` / `1` or `0`.
-- `device`: ordinal number of device on the bus. Range is from `0` to `n - 1`, where `n` is last number device. Use option value is `ALL` to selected and sets state of all power switches. If the `device` argument is omitted, it used only one board Octofet.
+- `channel`: the power switch index. Ranges from `0` to `7`.
+- `value`: defines the desired switch state. Valid values: `HIGH` / `1` / `true` to turn on or `LOW` / `0` / `false` to turn off.
+- `device`: the index of the affected Octofet in the daisy-chain. Ranges from `0` to `n - 1`, where `n` is the number of Octofets in the chain. If omitted, targets Octofet nearest to the controller.
 
 ### `void digitalWrite8(uint8_t value, uint8_t device = 0)`
 
-Sets state (turn-on or turn-off) for all power switches. Function arguments:
+Sets state (turn-on or turn-off) for all power switches at once.
 
-- `value`: 8-bit value for all of 8 power switches. One bit for one power switch: 1 is turn-on and 0 is turn-off.
-- `device`: ordinal number of device on the bus. Range is from `0` to `n - 1`, where `n` is last number device. Use option value is `ALL` to selected and sets state of all devices. If the `device` argument is omitted, it used only one board Octofet.
+- `value`: 8-bit value for all of 8 power switches. One bit for one channel: 1 is turn-on and 0 is turn-off. For example: `0b10101010`.
+- `device`: the index of the affected Octofet in the daisy-chain. Ranges from `0` to `n - 1`, where `n` is the number of Octofets in the chain. If omitted, targets Octofet nearest to the controller.
 
 ### `bool getChannelState(uint8_t channel, uint8_t device = 0)`
 
-Reads state (turn-on or turn-off) for one power switch of the device. Returns `true` or `false`. Function arguments:
+Returns the last set (i.e., current) state for one power switch of the device. Returns `true` if turned on or `false` otherwise.
```

Comment:
```suggestion
Returns the last (i.e., current) set state for one power switch of the device. Returns `true` if turned on or `false` otherwise.
```

Reply:
'cause last = current. Or it has to be "last set (i.e., current set)"

Reply:
:ok_hand:

Reply:
Done